### PR TITLE
Editorial: Export terms for payment method basic card spec

### LINKED ISSUE #890: Payment Method Basic Card spec uses various Payment Request definitions that Payment Request does not yet export
The payment method basic card spec has many `data-cite` attributes that link to various definitions in payment request. However, those definitions are not yet exported to xref. Could we export this to xref so that basic card can use xref for linking? Thanks! 

1. [Payment Request Options](https://github.com/w3c/payment-method-basic-card/blob/df6b49d757bdae56fb5197a40085cc53ddfcd95e/index.html#L299-L301)
2. [Payment Request Creating a Payment Address from user-provided input](https://github.com/w3c/payment-method-basic-card/blob/df6b49d757bdae56fb5197a40085cc53ddfcd95e/index.html#L311-L312)
3. [Payment Request Payment Method Changed Algorithm](https://github.com/w3c/payment-method-basic-card/blob/df6b49d757bdae56fb5197a40085cc53ddfcd95e/index.html#L423-L424)
4. [Payment Request serialized method data](https://github.com/w3c/payment-method-basic-card/blob/df6b49d757bdae56fb5197a40085cc53ddfcd95e/index.html#L466)
5. [Payment Request serialised modifier data](https://github.com/w3c/payment-method-basic-card/blob/df6b49d757bdae56fb5197a40085cc53ddfcd95e/index.html#L552)
6. [Payment Request Details](https://github.com/w3c/payment-method-basic-card/blob/df6b49d757bdae56fb5197a40085cc53ddfcd95e/index.html#L588-L589)
7. [Payment Request Payment Details Base Modifiers](https://github.com/w3c/payment-method-basic-card/blob/df6b49d757bdae56fb5197a40085cc53ddfcd95e/index.html#L589-L591)
8. [Payment Request Payment Details Modifier Total](https://github.com/w3c/payment-method-basic-card/blob/df6b49d757bdae56fb5197a40085cc53ddfcd95e/index.html#L593-L595)
9. [Payment Request Payment Details Modifier Additional Display Items](https://github.com/w3c/payment-method-basic-card/blob/df6b49d757bdae56fb5197a40085cc53ddfcd95e/index.html#L597-L600)

### PR DESCRIPTION
**Pull request by janiceshiu**
Closes https://github.com/w3c/payment-request/issues/890

cc @marcoscaceres

***
<a href="https://pr-preview.s3.amazonaws.com/janiceshiu/payment-request/pull/894.html" title="Last updated on Jan 23, 2020, 9:10 AM UTC (a77427e)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/payment-request/894/fe208db...janiceshiu:a77427e.html" title="Last updated on Jan 23, 2020, 9:10 AM UTC (a77427e)">Diff</a>

### CODE REVIEW

**Comment by ianbjacobs** [comment]
Marked as non substantive for IPR from ash-nazg.

File: index.html [hunk 1]
```diff
@@ -1443,7 +1443,7 @@ <h2>
           </tr>
           <tr>
             <td>
-              <dfn>[[\serializedMethodData]]</dfn>
+              <dfn data-export="">[[\serializedMethodData]]</dfn>
```
Comment (marcoscaceres) [inline index.html]:
```suggestion
              <dfn data-export="" data-dfn-for="PaymentRequest">[[\serializedMethodData]]</dfn>
```

Comment (marcoscaceres) [reply index.html]: I'm not sure if all of these need to be exported or not... are they all accessed by the basic card spec?

Comment (janiceshiu) [reply index.html]: @marcoscaceres yes, they are. the issue I raised [here](https://github.com/w3c/payment-request/issues/890) links to the parts of basic card spec that link to these terms. I double-checked by searching for `data-cite` in [this branch](https://github.com/w3c/payment-method-basic-card/pull/84) to make sure they're still there.

Comment (marcoscaceres) [reply index.html]: ok, thanks for confirming.

File: index.html [hunk 2]
```diff
@@ -1453,7 +1453,7 @@ <h2>
           </tr>
           <tr>
             <td>
-              <dfn>[[\serializedModifierData]]</dfn>
+              <dfn data-export="">[[\serializedModifierData]]</dfn>
```
Comment (marcoscaceres) [inline index.html]:
```suggestion
              <dfn data-export="" data-dfn-for="PaymentRequest">[[\serializedModifierData]]</dfn>
```

File: index.html [hunk 3]
```diff
@@ -1465,7 +1465,7 @@ <h2>
           </tr>
           <tr>
             <td>
-              <dfn>[[\details]]</dfn>
+              <dfn data-export="">[[\details]]</dfn>
```
Comment (marcoscaceres) [inline index.html]:
```suggestion
              <dfn data-export="" data-dfn-for="PaymentRequest">[[\details]]</dfn>
```

File: index.html [hunk 4]
```diff
@@ -1481,7 +1481,7 @@ <h2>
           </tr>
           <tr>
             <td>
-              <dfn>[[\options]]</dfn>
+              <dfn data-export="">[[\options]]</dfn>
```
Comment (marcoscaceres) [inline index.html]:
```suggestion
              <dfn data-export="" data-dfn-for="PaymentRequest">[[\options]]</dfn>
```

File: index.html [hunk 5]
```diff
@@ -4360,7 +4360,7 @@ <h2>
           Payment method changed algorithm
         </h2>
         <p data-cite="WEBIDL">
-          A <a>payment handler</a> MAY run the <dfn>payment method changed
+          A <a>payment handler</a> MAY run the <dfn data-export="">payment method changed
```
Comment (marcoscaceres) [inline index.html]:
```suggestion
          A <a>payment handler</a> MAY run the <dfn data-export="" data-dfn-for="payment handler">payment method changed
```

File: index.html [hunk 6]
```diff
@@ -1453,19 +1455,22 @@ <h2>
...
+              <dfn data-export="" data-dfn-for=
+              "PaymentRequest">[[\serializedModifierData]]</dfn>
             </td>
             <td>
               A list containing the serialized string form of each <a data-lt=
               "PaymentDetailsModifier.data">data</a> member for each
               corresponding item in the sequence
-              <a>[[\details]]</a>.<a data-lt="PaymentDetailsBase">modifier</a>,
-              or null if no such member was present.
+              [=PaymentRequest/[[details]]=].<a data-lt=
```
Comment (marcoscaceres) [inline index.html:1465]: We can probably fix this one: {{PaymentDetailsBase/modifiers}} ... might need to look this one up in the other spec to see if this is correct.

Comment (marcoscaceres) [reply index.html:1465]: oops, I mean in this spec.

File: index.html [hunk 7]
```diff
@@ -4530,9 +4542,9 @@ <h2>
           </li>
           <li>If the <a data-lt=
           "PaymentOptions.requestShipping">requestShipping</a> value of
-          |request|.<a>[[\options]]</a> is true, then if the <a data-lt=
-          "PaymentRequest.shippingAddress">shippingAddress</a> attribute of
-          |request| is null or if the <a data-lt=
+          |request|.[=PaymentRequest/[[options]]=] is true, then if the
+          <a data-lt="PaymentRequest.shippingAddress">shippingAddress</a>
```
Comment (marcoscaceres) [inline index.html:4546]:
```suggestion
          {{PaymentRequest/shippingAddress}}
```

Could you check if there are others like this one? We should kill any data-lt="Foo.Bar" ones.

File: index.html (hunk 6 above)
Comment (janiceshiu) [reply index.html:1465]: as discussed - gonna do this in a separate PR cos this PR is just for exporting terms. also, there are 150+ `data-lt` things. XD

File: index.html (hunk 7 above)
Comment (janiceshiu) [reply index.html:4546]: same as https://github.com/w3c/payment-request/pull/894#discussion_r370455253 ^^

**Review by marcoscaceres — APPROVED** [review]

**Pull request merged**